frontend/src/scenes/exports/api.ts

```diff
@@ -131,6 +132,37 @@ export const useCreateExport = (): {
     return { createExport, ...state }
 }
 
+export const useExportAction = (
+    teamId: number,
+    exportId: string,
+    action: string
```

Comment:
Perhaps this makes it clear what actions are available:

```suggestion
    action: 'pause' | 'unpause'
```

Reply:
Yeah, good catch. We can later add `'delete'` and `'backfill'`.